example.py

```diff
@@ -0,0 +1,25 @@
+import json
+import requests
+
+def revert_risk_api_request(language, revision_id):
+    use_auth = False
+    inference_url = 'https://api.wikimedia.org/service/lw/inference/v1/models/' + language + '-reverted:predict'
```

Comment:
[f-strings ](https://docs.python.org/3/reference/lexical_analysis.html#f-strings) is a nice way to build python strings when we want to render variables in them
```suggestion
    inference_url = f"https://api.wikimedia.org/service/lw/inference/v1/models/{language}-reverted:predict"
```